specification.yml

```
@@ -4551,6 +4566,41 @@ title: 'slicing object'
 context: {key: {x: 10}}
 template: {$eval: 'key[2:]'}
 error: true
+---
+title: 'string slicing - full'
+context: {key: "hello"}
+template: {$eval: 'key[0:6]'}
+result: "hello"
+---
+title: 'string slicing - prefix'
+context: {key: "hello"}
+template: {$eval: 'key[0:3]'}
+result: "hel"
+---
+title: 'string slicing - omitted first index'
+context: {key: "hello"}
+template: {$eval: 'key[:3]'}
+result: "hel"
+---
+title: 'string slicing - omitted second index'
+context: {key: "hello"}
+template: {$eval: 'key[3:]'}
+result: "lo"
+---
+title: 'string slicing - negative index'
+context: {key: "hello"}
+template: {$eval: 'key[-3:]'}
+result: "llo"
+---
```

Didn't look like you had a test like this.

```suggestion
title: 'string slicing - negative index, omitted second index'
context: {key: "hello"}
template: {$eval: 'key[-3:]'}
result: "llo"
---
title: 'string slicing - negative index'
context: {key: "hello"}
template: {$eval: 'key[-3:4]'}
result: "ll"
---
```